src/cephadm/cephadm

```diff
@@ -4788,21 +4788,21 @@ def command_check_host():
             find_program(command)
             logger.info('%s is present' % command)
         except ValueError:
-            errors.append('ERROR: %s binary does not appear to be installed' % command)
+            errors.append('%s binary does not appear to be installed' % command)
 
     # check for configured+running chronyd or ntp
     if not check_time_sync():
-        errors.append('ERROR: No time synchronization is active')
+        errors.append('No time synchronization is active')
 
     if 'expect_hostname' in args and args.expect_hostname:
         if get_hostname().lower() != args.expect_hostname.lower():
-            errors.append('ERROR: hostname "%s" does not match expected hostname "%s"' % (
+            errors.append('hostname "%s" does not match expected hostname "%s"' % (
                 get_hostname(), args.expect_hostname))
         logger.info('Hostname "%s" matches what is expected.',
                     args.expect_hostname)
 
     if errors:
-        raise Error('\n'.join(errors))
+        raise Error('\nERROR: '.join(errors))
```

Comment:
I see ok the output produced ( I do not see the effect pointed by @sebastian-philipp )

Just improve little bit using plural or singular if it is needed:
```python
If errors:
  error_message_header = "Errors:\n" if len(errors)> 1 else "Error:\n"
  raise Error(f'{error_message_header.join(errors)})
```

Reply:
I think that will still cause issues for the mgr/cephadm logic @sebastian-philipp referenced:
https://github.com/ceph/ceph/blob/beb00bd2a8ecc486fc6ddbe00cf9e4cb4bc20509/src/pybind/mgr/cephadm/module.py#L1192